data/jobs.txt

```diff
@@ -1573,3 +1573,192 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Dump garbage"
+	name "Dump garbage"
+	description "<origin> produces a lot of garbage that needs to be disposed of off-station. Transport <tons> of it to the industrial solid waste incinerator on <destination>. Payment is <payment>.
+	job
+	repeat
+	cargo "Heavy Metals" 15 100
+	source
+		attributes station
+	destination
+		attributes factory mining "dirt belt"
+		distance 10
+	on complete
+		dialog "You dump the garbage at an especially smelly solid waste incinerator and collect your payment of <payment>."
+		payment 20000 200
+
+mission "Provide security for pleasure cruise"
+	name "Provide security for pleasure cruise"
+	description "Escort the <npc> on a pleasure cruise to the wild and exotic planet of <stopovers> along with <tons> of their <commodity>, then return to <destination>, where you will be paid <payment>."
+	job
+	repeat
+	cargo "Luxury Goods" 10 20
+	to offer
+		random < 50
+		"combat rating" > 15
+	source
+		attributes paradise "near earth" rich
+	stopover
+		attributes tourism religious quarg pirate volcanic frontier moon fishing
+	destination
+		attributes paradise "near earth" rich core
+	npc accompany save
+		government Merchant
+		personality escort timid
+		fleet
+			names "civilian"
+			variant
+				"Star Queen"
+	on stopover
+		dialog "The passengers of the <npc> wander off, bedecked in the most conspicuous tourist attire they seem to have been able to find, and begin oohing and aahing at the exoticism of this faraway world. Hopefully none of them gets robbed. You prepare for the return journey to <destination>."
+	on visit
+		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+	on complete
+		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
+		payment 130000 500
+
+mission "Transport scientists"
```

Comment:
nice mission, future suggestion would be to have science vessels not just travel from planet to planet but for example venture out to study natural phenomena like stars or wormholes, and you need to guard them for a certain period of time while they carry out research.